corc-core/src/main/java/com/hotels/corc/mapred/CorcInputFormat.java

```diff
@@ -46,6 +54,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.apache.commons.codec.binary.Base64;
```

Comment:
These imports should be above where the other Apache imports are. Talk to one of the other IntelliJ users and see how they set things up to get the import order correct.